src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -41,11 +45,17 @@ public OverseasEntitySubmissionDtoValidator(EntityDtoValidator entityDtoValidato
     public Errors validate(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
         entityDtoValidator.validate(overseasEntitySubmissionDto.getEntity(), errors, loggingContext);
         presenterDtoValidator.validate(overseasEntitySubmissionDto.getPresenter(), errors, loggingContext);
-        if(Objects.nonNull(overseasEntitySubmissionDto.getOverseasEntityDueDiligence())) {
-            overseasEntityDueDiligenceValidator.validate(overseasEntitySubmissionDto.getOverseasEntityDueDiligence(), errors, loggingContext);
-        }
-        if(Objects.nonNull(overseasEntitySubmissionDto.getDueDiligence())) {
-            dueDiligenceValidator.validate(overseasEntitySubmissionDto.getDueDiligence(), errors, loggingContext);
+
+        DueDiligenceDto dueDiligenceDto = overseasEntitySubmissionDto.getDueDiligence();
+        OverseasEntityDueDiligenceDto overseasEntityDueDiligenceDto = overseasEntitySubmissionDto.getOverseasEntityDueDiligence();
+        if (DueDiligenceOptionValidators.bothDueDiligenceOptionsNotSupplied(dueDiligenceDto, overseasEntityDueDiligenceDto, errors, loggingContext)) {
```

Comment:
See comment below with suggested class rename, but if adopted, this method name could become 'bothBlocksSupplied()' to improve readability of the code.

Reply:
Updated this, see what you think.

Reply:
New comment added so resolving this one. Suggest a minor change though.